portal/src/main/webapp/js/src/cbio-stat.js

```diff
@@ -1,14 +1,12 @@
-if (cbio === undefined) {
```

Comment:
We shouldn't remove initialization here. I notice you declare the cbio in cbio-stat.spec.js but we will not include cbio-stat.spec.js in the final js build file, right?